plugins/inputs/t128_peer_path/README.md

```diff
@@ -0,0 +1,24 @@
+# 128T Peer Path Input Plugin
+
+The peer path input plugin collects data from a 128T instance via graphQL.
```

Comment:
This is a public doc so we need to boost it up. Can lift the blurb from [here](https://docs.128technology.com/docs/plugin_monitoring_agent#peer-path-state-collector)

> The peer path input can be used for monitoring the up/down status of all the peer paths on the node